subprojects/consumer/.gitignore

```diff
@@ -0,0 +1,4 @@
+.idea/
+.gradle/
+build/
+local.properties
```

Comment:
There are no local.properties in this dir.
Could be please remove it?
Or why have you added it?

```suggestion
```

Reply:
It will be created automatically by IntelliJ when opening the consumable project because it detects android modules.

Example content:

```
## This file must *NOT* be checked into Version Control Systems,
# as it contains information specific to your local configuration.
#
# Location of the SDK. This is only used by Gradle.
# For customization when using a Version Control System, please read the
# header note.
#Tue Oct 23 21:42:16 CEST 2018
ndk.dir=/Users/pascalwelsch/android-sdk/ndk-bundle
sdk.dir=/Users/pascalwelsch/android-sdk
```